src/components/organisms/SettingsManager/Settings.tsx

```diff
@@ -63,8 +63,8 @@ export const Settings = ({
 
   const resourceRefsProcessingOptions = useAppSelector(state => state.main.resourceRefsProcessingOptions);
   const uiState = useAppSelector(state => state.ui);
-  const appConfig = useAppSelector(state => state.config);
-  const fileMap = useAppSelector(state => state.main.fileMap);
+  const {isScanIncludesUpdated, isScanExcludesUpdated} = useAppSelector(state => state.config);
+  const filePath = useAppSelector(state => state.main.fileMap[ROOT_FILE_ENTRY].filePath);
```

Comment:
could this throw an error if you open the Settings drawer without having a project open?
better to write it as: `state.main.fileMap[ROOT_FILE_ENTRY]?.filePath`

Reply:
tested it and it does not but it does not hurt to have this in there